<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreestyleUI</class>
 <widget class="QWidget" name="FreestyleUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="leftMainFrame">
     <property name="minimumSize">
      <size>
       <width>235</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>235</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(156, 156, 116);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="leftMainWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item alignment="Qt::AlignmentFlag::AlignTop">
          <widget class="QFrame" name="leftSubFrame">
           <property name="frameShape">
            <enum>QFrame::Shape::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QFrame" name="infoFrame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="infoLabel">
                 <property name="text">
                  <string>New Labels Are To Be Added</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addLabel_pushbutton">
              <property name="text">
               <string>Add New Line</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="itemsNumber_pushbutton">
              <property name="text">
               <string>Number of Cells In Table</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rowCount_pushButton">
              <property name="text">
               <string>Row Count</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="columnCount_pushButton">
              <property name="text">
               <string>Column Count</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="rightMainFrame">
     <property name="minimumSize">
      <size>
       <width>447</width>
       <height>448</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(105, 105, 105);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QFrame" name="tableKeysFrame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="tableKeysGridLaout">
           <item row="0" column="5">
            <widget class="QLabel" name="regNoLabel">
             <property name="minimumSize">
              <size>
               <width>105</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <property name="text">
              <string>Reg. Number</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="serialNoLabel">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Plain</enum>
             </property>
             <property name="text">
              <string>S/N</string>
             </property>
             <property name="indent">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="firstnameLabel">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <property name="text">
              <string>Firstname</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="middlenameLlabel">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <property name="text">
              <string>Middlename</string>
             </property>
             <property name="indent">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="surnameLabel">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <property name="text">
              <string>Surname</string>
             </property>
             <property name="indent">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="tableValueFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="tableValuesGridLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
